edx/programs
# ECOM-3950 clean up stale copies of program banner images

@rlucioni @jimabramson Please review
This is the AC#1 of the ticket
Tested on django admin:
1. Upload a new banner image. Make sure it succeeds
2. Upload a banner image on top of existing banner image, make sure it succeeds and both images set are kept
3. Upload banner images on the same program multiple times and make sure the save succeeds and only 2 set of most recent images are kept
